java/core/src/main/java/com/microsoft/bond/BondSerializable.java

```diff
@@ -3,99 +3,16 @@
 
 package com.microsoft.bond;
 
-import com.microsoft.bond.protocol.ProtocolWriter;
-import com.microsoft.bond.protocol.TaggedProtocolReader;
-
-import java.io.IOException;
-import java.io.InputStream;
-
-
 /**
- * Enables serialization of Bond classes.
+ * Denotes a Bond enumeration type.
```

Comment:
Yes, Chad already pointed this out. It was a copy-paste.

---
In reply to: [123116280](https://github.com/Microsoft/bond/pull/502#discussion_r123116280) [](ancestors = 123116280)